### PR TITLE
First Test instance of a class

### PR DESCRIPTION
**Pull request by zahodi**
- gitignore __pycache__ and move bovine.py to lib/

### CODE REVIEW

**Review by senorsmile — DISMISSED** [review]
wouldn't lib/bovine/bovine.py make the module be named:

bovine.bovine

we probably just want 
./bin/bovine.py

so that it is simply call bovine

and importable with

import bovine

**Comment by zahodi** [comment]
I will actually rename bovine.py to what it is actually doing.

**Comment by zahodi** [comment]
Review Criteria:
- Verify that both Travis checks are green for the latest commit.

File: lib/bovine/staticinventory.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 ##########################
 
 
-class Inventory:
+class GatherStaticInventory:
```
Comment (senorsmile) [inline lib/bovine/staticinventory.py:9]: We will want to rename this.  GatherStaticInventory sounds more like a method name (e.g. VerbObject).  Class names should be a 'class' of things... we will probably want the Class name to either be
- Inventory
of
- StaticInventory

I suppose StaticInventory could be a subclass of Inventory.  

Don't make any changes for now though.  This looks good for initial tests.

File: test/context.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+import sys
+import os
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```
Comment (senorsmile) [inline test/context.py:4]: this would be way more legible on multiple lines... think FP (functional programming).

**Review by senorsmile — APPROVED** [review]
lgtm 👍

File: test/context.py (hunk 2 above)
Comment (senorsmile) [reply test/context.py:4]: Also confirmed all tests passed in travis.  woo hoo!

**Pull request merged**